lib/Inspection/include/Inspection/Transformers.h

```diff
@@ -26,21 +26,34 @@
 #include <chrono>
 #include "Basics/TimeString.h"
 #include "Basics/ErrorCode.h"
+#include "Inspection/Status.h"
+
+#include "date/date.h"
+#include "fmt/core.h"
 
 namespace arangodb::inspection {
 
 struct TimeStampTransformer {
   using SerializedType = std::string;
   using clock = std::chrono::system_clock;
+  static constexpr std::string_view formatString = "%Y-%m-%d %H:%M:%S";
```

Comment:
Shouldn't we format this as UTC ISO8601? In that case the format string should look like this:
```suggestion
  static constexpr std::string_view formatString = "%Y-%m-%dT%H:%M:%SZ";
```